vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
Because  https://github.com/rancher/os/blob/master/trash.conf contains `github.com/mattn/go-shellwords v1.0.0`  line 32 which updates https://github.com/tetatetit/os/tree/master/vendor/github.com/mattn/go-shellwords on each `trash` run to the latest https://github.com/mattn/go-shellwords where https://github.com/mattn/go-shellwords/blob/master/.travis.yml does not contain `- go get github.com/axw/gocov/gocov` appropriately. So I not sure why to keep this line in our repo while it is not in original repo from where trash updates it.